### DISCUSSION TITLE
Measuring how many events a cluster collector is processing

### DISCUSSION THREAD

**Discussion by DevinTyler26** [open-telemetry/opentelemetry-collector#5724] [Q&A]
Fundamentally, we want the unit of measurement to be "events" where an event is a signal (metric, trace, log, etc). We care about the end to end processing of an event, and we care about how many events per second we can handle. And we want to export this data to a sync like DataDog to create an SLO.
Is there a way to see how many events (metric, trace, log, etc) an OTEL cluster collector is processing?

**Comment by tigrannajaryan** [comment]
Collector exposes certain metrics. See if these provide the information you need: https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/troubleshooting.md#metrics
